### PR TITLE
User profile improvements

### PR DESCRIPTION
**Pull request by jorolf**
- [x] Depends on https://github.com/ppy/osu-framework/pull/907

### CODE REVIEW

File: osu.Game/Overlays/Profile/ProfileHeader.cs [hunk 1]
```diff
@@ -380,9 +383,9 @@ private void loadUser()
             tryAddInfoRightLine(FontAwesome.fa_suitcase, user.Occupation);
             infoTextRight.NewParagraph();
             if (!string.IsNullOrEmpty(user.Twitter))
-                tryAddInfoRightLine(FontAwesome.fa_twitter, "@" + user.Twitter);
-            tryAddInfoRightLine(FontAwesome.fa_globe, user.Website);
-            tryAddInfoRightLine(FontAwesome.fa_skype, user.Skype);
+                tryAddInfoRightLine(FontAwesome.fa_twitter, "@" + user.Twitter, @"http://twitter.com/" + user.Twitter);
+            tryAddInfoRightLine(FontAwesome.fa_globe, user.Website, user.Website);
+            tryAddInfoRightLine(FontAwesome.fa_skype, user.Skype, @"skype:" + user.Skype + @"?chat");
```
Comment (Nabile-Rahmani) [inline osu.Game/Overlays/Profile/ProfileHeader.cs]: Is that supposed to be a protocol ? In which case you're probably missing `//`.

Comment (jorolf) [inline osu.Game/Overlays/Profile/ProfileHeader.cs]: well, osu-web uses the same format and it seems to be working fine for me

Comment (jorolf) [inline osu.Game/Overlays/Profile/ProfileHeader.cs]: image

Comment (Nabile-Rahmani) [inline osu.Game/Overlays/Profile/ProfileHeader.cs]: Does that work if you input that in the Run window (`Win+R`) ?

Comment (jorolf) [inline osu.Game/Overlays/Profile/ProfileHeader.cs]: yes

Comment (jorolf) [inline osu.Game/Overlays/Profile/ProfileHeader.cs]: why don't you try yourself?

Comment (Nabile-Rahmani) [inline osu.Game/Overlays/Profile/ProfileHeader.cs]: I have neither Skype or Windows.

**Comment by peppy** [comment]
Please avoid commits like "minor improvements". You make three or four different distinct changes which deserve their own commits.

**Comment by peppy** [comment]
Now depends on https://github.com/ppy/osu-framework/pull/907

File: osu.Game/Overlays/Profile/ProfileHeader.cs [hunk 2]
```diff
@@ -379,13 +379,21 @@ private void loadUser()
                 infoTextLeft.AddText(string.Join(", ", user.PlayStyle), boldItalic);
             }
 
+            string websiteWithoutProtcol = user.Website;
```
Comment (jorolf) [inline osu.Game/Overlays/Profile/ProfileHeader.cs]: Protcol?

**Comment by jorolf** [comment]
pretty sure it doesn't depend on https://github.com/ppy/osu-framework/pull/907 anymore

**Comment by peppy** [comment]
It does, as that still fixes one of the issues I wanted to fix.

**Comment by jorolf** [comment]
image this needs to be fixed too :/

**Comment by peppy** [comment]
Blocked by https://github.com/ppy/osu/pull/1037

**Review by peppy — APPROVED** [review]

**Pull request merged**